jerry-core/ecma/operations/ecma-promise-object.h

```diff
@@ -34,9 +34,8 @@ typedef enum
   ECMA_PROMISE_IS_PENDING = (1 << 0), /**< pending state */
   ECMA_PROMISE_IS_FULFILLED = (1 << 1), /**< fulfilled state */
   ECMA_PROMISE_ALREADY_RESOLVED = (1 << 2), /**< already resolved */
-  ECMA_PROMISE_HANDLED = (1 << 3), /**< ES11: 25.6.6 [[PromiseIsHandled]] internal slot */
 #if JERRY_PROMISE_CALLBACK
-  ECMA_PROMISE_UNHANDLED_REJECT = (1 << 4), /**< a Promise is rejected without a catch handler */
+  ECMA_PROMISE_UNHANDLED_REJECT = (1 << 3), /**< a Promise is rejected without a catch handler */
```

Comment:
So this bit is used for the `[[PromiseIsHandled]]` ? Could we add the ES reference into the comment? So we can grep for if someone looks for it.